packages/matchbox/src/components/Layout/Layout.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Columns } from '../Columns';
+
+import Section from './Section';
+import SectionTitle from './SectionTitle';
+
+const breakpoints = ['xs', 'sm', 'md', 'lg', 'xl'];
+
+const Layout = React.forwardRef(function Layout(props, ref) {
+  const { children, collapseBelow } = props;
+
+  return (
+    <Columns collapseBelow={collapseBelow} space="500" mb="36px" ref={ref}>
```

Comment:
Can we add a comment here about `36` not being a token - and replaces with `2.25rem`

Reply:
Or just replace with `600` for now, will bring it up in standup tomorrow